study.md

```diff
@@ -22,5 +22,15 @@ Ember](https://www.codeschool.com/courses/try-ember) course from CodeSchool.
 In your own words, describe why someone would use Ember.
 
 ```md
-<!-- your answer here -->
+Ember streamlines the creation of single-page apps by conventionalizing the
+development process according to industry best practices. This makes code easier
+to create, read, and edit. Similarly to Rails, Ember can quickly generate models,
+routes, controllers, serializers, and other high-level tools for creating SPAs.
+Handlebars is built-in, so using and presenting server data is relatively simple
```

Comment:
Technically it's a sub-set of Handlebars, called HTMLbars. But it's basically the same.